pkg/scheduler/partition_test.go

```diff
@@ -1828,6 +1839,8 @@ func TestPreemption(t *testing.T) {
 	// delay so that preemption delay passes
 	time.Sleep(100 * time.Millisecond)
 
+	time.Sleep(3 * time.Second)
```

Comment:
Oh, sorry, I thought I responded to this question earlier ...

This is because we are using resource * serconds to record resource usage, so at least we need one second, I put 3 randomly.

It's a good idea to try something like "Tick()", but that requires changing all time.Now() to be replaced with the interface you suggested?

Thanks.

Reply:
For now just use Sleep(). We can do that in a separate JIRA.